lab-matt/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This is a very well made package.json, I really like that you are using a "start" command as well as a "test" command.  There was one minor issue, see my comment below.  Generally though keep up the great work!